src/mscorlib/shared/System/Globalization/CompareInfo.cs

```diff
@@ -74,9 +74,9 @@ public partial class CompareInfo : IDeserializationCallback
         [OptionalField(VersionAdded = 3)]
         private SortVersion m_SortVersion; // Do not rename (binary serialization)
 
-        // _invariantMode is defined for the perf reason as accessing the instance field is faster than access the static property GlobalizationMode.Invariant
+        // s_invariantMode is defined for the perf reason as accessing the field is faster than access the static property GlobalizationMode.Invariant
         [NonSerialized]
-        private readonly bool _invariantMode = GlobalizationMode.Invariant;
```

Comment:
GlobalizationMode.Invariat is a readonly static already. We do not need to duplicate it.

Reply:
This is cached in a instance field because instance field access is faster (in crossgened code in particular - and CoreLib is pretty much always crossgened).

Reply:
right, we just need to replace GlobalizationMode.Invariant with _invariantMode in CompareInfo code.

Reply:
All the instance methods already use it; other ones are static